src/main/java/com/salesforce/mirus/OffsetStatus.java

```diff
@@ -75,7 +75,7 @@ private void run() {
           .forEach(
               k ->
                   System.out.printf(
-                      "%s\t%d\t%d\t%d\n",
+                      "%s\t%d\t%d\t%d" + System.lineSeparator(),
```

Comment:
```suggestion
                      "%s\t%d\t%d\t%d%n",
```

Reply:
Can probably change the other ones in `OffsetSerDeTest` to `%n` as well maybe :smile:

Reply:
Only works with `printf` and friends unfortunately.